src/stories/Library/link-with-icon/link-with-icon.scss

```diff
@@ -0,0 +1,60 @@
+.link-with-icon {
+  background-color: $c-global-primary;
+  padding: $s-md $s-lg;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  border: 1px solid $c-global-tertiary-1;
+  line-height: 160%;
+  width: 100%;
+  @include container(medium);
+
+  > svg {
+    display: block;
+    width: 35px;
+    margin-left: auto;
+    visibility: hidden;
+  }
```

Comment:
Please reconsider this.

As far as I can tell the intention is to target the `arrow` child element. In general we should not do so this way. In general: Without using classes or code comments it is hard to tell what is going on and why.

On the other hand I can see that [we seem to practice this in other situations](https://github.com/danskernesdigitalebibliotek/dpl-design-system/blob/develop/src/stories/Library/card-list-item/card-list-item.scss#L23) and you might have been inspired from them. If so then I would at least like us to use the mixin provided by `arrows.scss`.